packages/__tests__/validate-package-json.spec.ts

```diff
@@ -0,0 +1,30 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import util from 'util';
+import globCb from 'glob';
+import fsCb from 'fs';
+
+const glob = util.promisify(globCb);
+const readFile = util.promisify(fsCb.readFile);
+
+describe('packages', () => {
```

Comment:
Feel free to move/refactor/etc this test. Without it, things are likely to get out of sync again.